src/models/subscription.ts

```diff
@@ -41,9 +41,10 @@ export class Subscription extends Model {
 	repositoryStatus?: TaskStatus;
 	gitHubAppId?: number;
 
-	static async getAllForHost(jiraHost: string): Promise<Subscription[]> {
+	static async getAllForHost(jiraHost: string, gitHubAppId?: number): Promise<Subscription[]> {
```

Comment:
Hey @krazziekay , can you please change it to `gitHubAppId: number | undefined`?

Reply:
`getAllForHost` is already being used in a lot of places and if we define it like this, then the second parameter will be required. Thats why I made the second params as optional...